### PR TITLE
x11-themes/e-gtk-theme: new package

### PR DESCRIPTION
**Pull request by juippis**
Didn't want to overcomplicate things with USE flags. I feel like this is a cleaner way to only getting GTK theme, something else I can think of:
```
src_prepare() {
	default

	sed -i -e 's|/usr/local|/usr|g' \
		-e 's|$(apps_DATA) $(dist_EXTRA) $(theme_DATA) $(openbox_DATA)|$(theme_DATA)|g' \
		-e 's|install-apps install-doc install-theme|install-theme|g' \
		Makefile || die
}
```
(not tested but you get the idea)

### CODE REVIEW

**Comment by hanetzer** [comment]
@juippis I could probably coerce a meson build onto this. Can you gist a list of installed
files?

**Comment by juippis** [comment]
@hanetzer 
sounds good. Pinged you in IRC, I wanna learn more when we're both around to talk about it.

File: x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+DESCRIPTION="A GTK theme to match Enlightenment WM's default theme"
+HOMEPAGE="https://gitlab.com/tokiclover/e-gtk-theme"
+SRC_URI="https://gitlab.com/tokiclover/${PN}/-/archive/${PV}/${P}.tar.gz"
+
+LICENSE="BSD-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_compile() {
+	:;
```
Comment (mgorny) [inline x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild]: `;` is meaningless here (it's equivalent to newline).

File: x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+src_install() {
+	insinto /usr/share/themes/e-gtk-theme
+	doins -r gtk-2.0/
```
Comment (mgorny) [inline x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild]: Why not pass them all to a single `doins`?

File: x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+	doins -r gtk-3.0/
+	doins index.theme
+	doins start-here.png
+
+	einstalldocs
+
+	find "${D}" -name 'Makefile' -delete || die
```
Comment (mgorny) [inline x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild]: If the Makefiles really aren't useful, why not remove them earlier (say, in `src_prepare`)? Then you wouldn't have to override phases.

File: x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+pkg_postinst() {
+	elog "If you wish to have matching theme for Openbox or Firefox, visit"
+	elog "project's homepage at https://gitlab.com/tokiclover/e-gtk-theme"
```
Comment (mgorny) [inline x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild]: Will visiting the homepage make it magically happen? ;-)

Comment (juippis) [reply x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild]: Hmm, I guess just install everything unconditionally then? Indeed they are "part of the package", and small files. 

I'll add the enhancements from above too soonish.

Comment (mgorny) [reply x11-themes/e-gtk-theme/e-gtk-theme-0.19.3.ebuild]: I just meant the message sounds a bit wrong.

**Pull request closed without merging**